tests/env.go

```diff
@@ -37,6 +40,8 @@ type TestEnv struct {
 	PublicTenantId       string
 	MerkelogURLPrefix    string
 	PublicKey            string
+
+	UknownTenantId string
```

Comment:
Unknown? or Uknown? if the latter what does the U mean? or is it meant to be KnownTenantd?

Reply:
It's meant to be "a tenant uuid which does not match any known tenant" So Unkown